src/lib/elements/forms/inputTags.svelte

```diff
@@ -89,7 +89,7 @@
                                     type="button"
                                     class="input-tag-delete-button"
                                     aria-label={`delete ${tag} tag`}
-                                    on:click={() => removeValue(tag)}>
+                                    on:click={() => !readonly && removeValue(tag)}>
```

Comment:
I think the `readonly` check should be in the `removeValue` function. Otherwise you can remove tags with the backspace key 👍🏻 

https://github.com/appwrite/console/blob/24b8e2221107aac2db73a7ac1fb2c10607d90488/src/lib/elements/forms/inputTags.svelte#L53-L55

```
if (readonly) return;
```
should to the trick 👍🏻

Reply:
@TorstenDittmann thank you for the suggestion. Have done the change in code